src/components/GameField/GameField.css

```diff
@@ -0,0 +1,33 @@
+.game-board {
+  margin: auto;
+  width: 600px;
+  height: 600px;
+  box-sizing: content-box;
+  border: 4px solid cadetblue;
+}
+
+.main-field {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  height: 600px;
+}
+
+.field {
+  height: 40px;
+  width: 40px;
+  border: 1px solid black;
+  box-sizing: border-box;
+}
+
+.green {
+  background-color: rgb(39, 221, 39);
```

Comment:
Consider moving the colors to CSS variables and changing class names to something like "odd" and "even"